rest_authenticate/tests/test_rest_session.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 Wakari SRL
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import odoo.tests.common
+from odoo.http import root
+from odoo.tests import new_test_user
+from odoo.tests.common import HttpCase
+
+HOST = "127.0.0.1"
+PORT = odoo.tools.config["http_port"]
+
+
+@odoo.tests.common.at_install(False)
```

Comment:
FYI you can now `@tagged` to set this flags

Reply:
Great! Thanks, much shorter